web/src/app/modules/views/main/editors/modules/contents-container/components/ceg-model-container.component.ts

```diff
@@ -31,21 +32,39 @@ export class CEGModelContainer extends TestSpecificationContentContainerBase<CEG
     }
 
     modelDescription: string;
+    objectifIndicator = 'WENN';
 
+    public isObjectif(): boolean {
+        let desc = this.modelDescription;
+
+        if (desc == undefined) {
+            desc = '';
+        }
+        desc = desc.trim();
+
+        if (desc.length <= 3) {
+            return false;
+        }
+        let relevantLength = Math.min(this.objectifIndicator.length, desc.length);
+        let cmpA = this.objectifIndicator.substring(0, relevantLength);
+        let cmpB = desc.substring(0, relevantLength);
+        return cmpA === cmpB;
+    }
 
     protected condition = (element: IContainer) => Type.is(element, CEGModel);
 
     public async createElement(name: string): Promise<CEGModel> {
         let factory: ModelFactoryBase = new CEGModelFactory(this.dataService);
         const element = await factory.create(this.parent, true, Id.uuid, name) as CEGModel;
 
-        if (this.modelDescription == undefined) {
-            this.modelDescription = '';
+        let description = this.modelDescription;
+        if (description == undefined) {
+            description = '';
         }
-        this.modelDescription = this.modelDescription.trim();
+        description = description.trim();
 
-        if (this.modelDescription.length > 0) {
-            element.modelRequirements = this.modelDescription;
+        if (description.length > 0) {

```

Comment:
please comment on what is happening here or better, create a separate method with a speaking name